decidim-admin/app/controllers/decidim/admin/scopes_controller.rb

```diff
@@ -79,6 +75,10 @@ def parent_scope
         @parent_scope ||= @scope ? @scope.parent : collection.find_by_id(params[:scope_id])
       end
 
+      def parent_scope_children
+        @parent_scope_children ||= parent_scope ? parent_scope.children : collection.top_level
+      end
+
```

Comment:
I find this name a bit misleading since it's only a "parent scope children" when there's a parent scope. Naming is hard... :sweat:. I'd rename `collection` to `organization_scopes`, and use `collection` (or maybe even `scopes_collection`, or `current_scopes`, or something like that) for this one.